packages/components/src/components/button/button.custom.styles.ts

```diff
@@ -1,5 +1,133 @@
 import { css } from 'lit';
 
 export default css`
-  /* Write custom CSS here */
-`;
+.button:focus-visible {
+    outline: var(--syn-focus-ring-color) solid 2px;
```

Comment:
I think the outline-width should also be a a design token so we can share it across other components, too. Example (naming to be defined!):

```css
outline: var(--syn-focus-ring-color) solid var(--syn-focus-ring-width);
outline-offset: var(--syn-focus-ring-offset);
```

Reply:
Good point. Let's do that in a follow-up ticket, so we can merge this soon and design can provide it then.

Reply:
Thats fine for me. Will you create the tickets for it?